modules/ROOT/pages/functions/scalar.adoc

```diff
@@ -1427,3 +1427,98 @@ The relationship type of `r` is returned.
 
 ======
 
+[[functions-valueType]]
+== valueType()
+
+The function `valueType()` returns a `STRING` representation of the most precise value type that the given expression evaluates to.
+
+*Syntax:*
+
+[source, syntax, role="noheader"]
+----
+valueType(expression)
+----
+
+*Returns:*
+
+|===
+
+| `STRING`
+
+|===
+
+*Arguments:*
+
+[options="header"]
+|===
+| Name | Description
+
+| `expression` | Any expression that returns a value.
+
+|===
+
+*Considerations:*
+
+It is possible that future releases of Cypher will include updates to the current type system.
+This can include the introduction of new types and subtypes of already supported types.
+If a new type is introduced, it will be returned by the valueType() function as soon as it is released.
+However, if a more precise subtype of a previously supported type is introduced, it would be considered a breaking change.
+As a result, any new subtypes introduced after the release of Neo4j 5.13 will not be returned by the valueType() function until the following major release (Neo4j 6.0).
+For example, the function currently returns `"FLOAT"`, but if a more specific `FLOAT` type was added, e.g. `FLOAT32` this would be considered more specific and not returned until Neo4j 6.0, and `"FLOAT"` would continue to be returned for any `FLOAT32` values.
```

Comment:
```suggestion
For example, the function currently returns `"FLOAT"`, but if a more specific `FLOAT` type was added, e.g. `FLOAT32`, this would be considered more specific and not returned until Neo4j 6.0.
As a result,`"FLOAT"` would continue to be returned for any `FLOAT32` values.
```